code/MUT/MUT/YMLMeister.cs

```diff
@@ -281,15 +289,25 @@ public static bool IsMultilineValue(string filedata, int start)
             return d;
         }
 #endif
-        public static List<Tag> ParseYML2(string filedata)
+        public static List<Tag> ParseYML2(string filedata, string tagToFind)
         {
             var yml = GetYmlBlock(filedata);
 #if CURIOUS
             Console.WriteLine("vvvvvvvvvvvvvvv YML vvvvvvvvvvvvvvvvvvv");
             Console.WriteLine(yml);
             Console.WriteLine("^^^^^^^^^^^^^^^ YML ^^^^^^^^^^^^^^^^^^^");
 #endif
-            var tags = GetAllTags(yml);
+            // user can specify in command line whether they just want to
+            // get the values for one tag across all documents
+            List<string> tags;
+            if (tagToFind != null && tagToFind.Length > 0)
```

Comment:
if (!String.IsNullOrEmpty(tagToFind)) is another way to say this, if you like that. Or you could switch the logic and do

if (String.IsNullOrEmpty(tagToFind))...GetAllTags
else...GetOneTag